src/features/hotspots/details/HotspotDetailChart.tsx

```diff
@@ -115,12 +116,22 @@ const HotspotDetailChart = ({
   const { black, grayLight, grayMain, purpleMain, greenOnline } = useColors()
   const [focusedData, setFocusedData] = useState<ChartData | null>(null)
 
-  const onFocus = useCallback((chartData: ChartData | null) => {
+  const onFocus = useCallback(async (chartData: ChartData | null) => {
     if (Platform.OS === 'ios') {
       // this animation causes layout issues on Android
       animateTransition()
     }
-    setFocusedData(chartData)
+
+    if (!chartData) {
+      setFocusedData(null)
+      return
+    }
+
+    const label = await DateModule.formatDate(
+      chartData?.label,
+      chartData.showTime ? 'MMMdh:mma' : 'MMMd',
```

Comment:
Hmm on Android I don't think the locale format is working correctly either. 

On Android I get `März24` on iOS I get `18. Mar`

Reply:
> Does the new formatter support day of week? It used to be `Wed, Mar 24` and now it just says `Mar 24` for the non time view

Yeah, I'll fix it.

Reply:
> Hmm on Android I don't think the locale format is working correctly either.
> 
> On Android I get `März24` on iOS I get `18. Mar`

Which locale is this for @tyler-whitman ?

Reply:
> I think something might be off with the timezones too. Currently at 10:55 AM PST its showing me 5:00 PM as the last bar in the HNT Rewards hourly chart. It should be 10:00 AM as it used to be.

I'm seeing this too. Will fix...

Reply:
> > Hmm on Android I don't think the locale format is working correctly either.
> > On Android I get `März24` on iOS I get `18. Mar`
> 
> Which locale is this for @tyler-whitman ?

This was for Deutsch (Deutchland)

Reply:
> > > Hmm on Android I don't think the locale format is working correctly either.
> > > On Android I get `März24` on iOS I get `18. Mar`
> > 
> > 
> > Which locale is this for @tyler-whitman ?
> 
> This was for Deutsch (Deutchland)

I did some playing around with this. On android my locale gets set to `de_DE` whereas on iOS my locale gets set to `en_DE` which may be because I have English as my second language?

Reply:
Suffice it to say, I think the code is right.